voxel51/apps/api.py

```diff
@@ -209,8 +209,8 @@ def upload_analytic_image(self, analytic_id, image_tar_path, image_type):
         mime_type = _get_mime_type(image_tar_path)
         with open(image_tar_path, "rb") as df:
             files = {"file": (filename, df, mime_type)}
-            res = self._requests.post(
-                endpoint, headers=self._header, files=files, params=params)
+            res = voxu.upload_files(self._requests, endpoint, files,
```

Comment:
nit: python style guide (https://github.com/voxel51/eta/blob/develop/docs/python_style_guide.md) says to not leave args on first line if breaking is required (unless you are going to match indent). So style says:

```py
res = voxu.upload_files(
    self._requests, endpoint, files, headers=self._header, params=params)
```

Reply:
Our JS style guide recommends this too (https://google.github.io/styleguide/jsguide.html#formatting-where-to-break), but I was not able to get it enforced in other repos